ballerina/vector_store.bal

```diff
@@ -29,28 +29,21 @@ public isolated class VectorStore {
     *ai:VectorStore;
     final postgresql:Client dbClient;
     private final int vectorDimension;
-    private string tableName = "vector_store";
+    private string tableName;
     private final ai:VectorStoreQueryMode embeddingType;
     private final SimilarityMetric similarityMetric;
 
     # Initializes the pgvector vector store with the provided configuration.
     #
     # + configs - Contains configuration for database connections and other necessary parameters
-    public isolated function init(Configuration configs) returns ai:Error? {
+    public isolated function init(string host, string user, string password, string database,
```

Comment:
shall we add the `@label` for each parameter. And we need to update the doc comments here

Reply:
Fixed by 1efeb01